docs: Remove index field from objectsReport in tracker response [DHIS2-15596] (#122)
dhis2/dhis2-releases: releases/2.41/README.md

* `/tracker/trackedEntities?attachment`
* `/tracker/event?attachment`

> The `index` field of an entity in the report of a tracker import was removed.

Reviewer: Should we create separate sections 🤔 So breaking changes to parameters and payload. We can also postpone re-structuring our notes closer to the release 👌🏻

Author: We definitely needs some restructuring but as you said it will be easier closer to the release

When importing tracker entities using `POST /tracker` endpoint, the response follow the format described [here](https://docs.dhis2.org/en/develop/using-the-api/dhis-core-version-master/tracker.html#import-summary-structure).

> In the `objectsReport` section the objects do not have anymore the `index` field as now they are ordered in the same way the request payload was ordered.

Reviewer: ```suggestion
The `index` field has been removed from the `objectsReport` as the objects are now ordered in the same way as in the request.
```
